academics/ImportantDates.md

```diff
@@ -4,7 +4,48 @@ title: Important Dates
 sidebar_label: Important Dates
 slug: /dates
 ---
-_Last updated on 2022/09/28_
+
+_Last updated on 2023/07/04_
+
+### Fall 2023
+
+-   July 10 - Registration Opens in UWinsite Student for RETURNING students - Fall 2023
+-   August 15 - Fees Due: Fall 2023
+-   September 4 - Labour Day: University Closed
+-   September 7 - First Day of Fall 2023 Classes
+-   September 20 - Academic Add/Drop for Fall 2023 courses
+-   October 4 - Financial Drop Date: Fall 2023
+-   October 7 to October 15 - Reading week
+-   October 9 - Thanksgiving Day: University Closed
+-   November 15 - Last day to voluntarily withdraw from courses
+-   December 6 - Last Day of Fall 2023 Classes
+-   December 9 to December 20 - Final Exams for Fall 2023
+-   December 21 - Alternate final exam date
+-   December 23 - January 2 - University Offices Closed for December Holiday recess
+
+### Summer 2023
+
+-   July 10 - Financial Drop Date - Summer 2023 (6-week)
+-   July 19 - Last Day to Voluntarily Withdraw from Full Summer 2023 (12 week) courses
+-   July 27 - Last Day to Voluntarily Withdraw from Summer Session 2023 (6 week) courses
+-   August 7 - Civic Holiday: University Closed
+-   August 8 - Last Day of Summer Session 2023 (6 week) Classes
+-   August 9 - Last Day of Full Summer 2023 (12 week) Classes
+-   August 12 to August 21 - Final Exams for Full Summer 2023 (12 week) and Summer (6 week) Session
+
+### Winter 2023
+
+-   January 5 - Classes begin
+-   February 1 - Full tuition refund (less non-refundable deposit if applicable) drop date for the Winter 2023 semester
+-   February 15 - Last day to submit a Full-Time OSAP application for the winter term
+-   February 18 to February 26 - Reading week
+-   February 20 - Family Day: University Closed
+-   February 24 - University Offices Closed
+-   March 4 - Last day to submit OSAP appeals and Part Time OSAP Application deadline for winter term
+-   April 5 - Last day of classes
+-   April 7 - Good Friday: University Closed
+-   April 10 to April 20 - Final exams
+-   April 21 - Alternate final exam date
 
 ### Fall 2022
```

Comment:
Do we want to keep Fall 2022 around? Its quite old at this point.

Reply:
I thought we wanted to keep everything and not delete, but sure we can delete and keep only 3 semesters at once

Reply:
Do the dates of 22 line up with the corresponding 23 semester?

Reply:
No, there is a day difference in a lot of the dates

Reply:
I'll leave it to your discretion then. Resolve this thread if you would rather keep them.

Reply:
@BorhanSaflo @Dante-Masciotra any thoughts?

Reply:
I think having an archive of important dates isn't a terrible idea but on this page is gonna get very long if every semester we just add to it. I prefer to keep it for 3 semester max

Reply:
Could create a sub-page or something and indicate that its an archive

Reply:
I like the idea of a sub-page to archive all the old ones and keep the "main" page clean and easy to read

Reply:
Lets add that in this PR.

Reply:
So it will be:

Important Dates 
 - Current
 - Archived

Reply:
Yes

Reply:
Is it possible to have the 3 current semesters on the Important Dates page, and then only Archive as a subpage? And maybe something like "Click [here](example.com) to see important dates for previous semesters"? So the layout would be:

Important Dates
- Archived